src/main/resources/db/migration/V043__fix_data.sql

```diff
@@ -0,0 +1,7 @@
+COPY public.search_serviceareacourt (servicearea_id, court_id, catchment_type) FROM stdin;
+2	1480143	national
```

Comment:
After a second look at this, I think the servicearea_id should be 5 and not 2

Reply:
agreed, fixed